internal/command/volumes/create.go

```diff
@@ -79,6 +80,43 @@ func runCreate(ctx context.Context) error {
 		return err
 	}
 
+	// If the Yes flag has been supplied we skip the warning entirely, so no
+	// need to query for the set of volumes
+	if !flag.GetYes(ctx) {
+		// fetch the set of volumes for this app. If > 0 we skip the prompt
+		var volumes []api.Volume
+		if volumes, err = client.GetVolumes(ctx, appName); err != nil {
+			return err
+		}
+
+		var matches int32
+		for _, volume := range volumes {
+			if volume.Name == volumeName {
+				matches++
+			}
+		}
+
+		if matches == 0 {
+			var confirmed bool
+			io := iostreams.FromContext(ctx)
+			colorize := io.ColorScheme()
+
+			const msg = "Warning! Individual volumes are pinned to individual hosts. You should create two or more volumes per application. You will have downtime if you only create one. If you proceed, make sure to create a second volume with fly volume create --require-unique-zone to ensure a second volume exists on a separate worker"
+			fmt.Fprintln(io.ErrOut, colorize.Red(msg))
+
+			switch confirmed, err = prompt.Confirm(ctx, "Do you still want to use the volumes feature?"); {
```

Comment:
Do we really want to bug the user with a confirmation after that long warning message? I can accept it but seems like we are overreacting :)

Reply:
I could go either way on the confirmation. It does emphasise the gravity of the decision. :)

This is such a complicated idea for a few words! They need to set up replication too, if they need data availability. Would it be easier to say: 

"Warning! Individual volumes are pinned to individual hosts. You should create two or more volumes per application. You will have downtime if you only create one. Learn more at https://fly.io/docs/reference/volumes/"

I know you couldn't have done this before because the messaging was tied up in the [apps v2 PR](https://github.com/superfly/docs/pull/627). I brought some over to main, to plug the gap.

Reply:
Linking to the docs makes a ton of sense - this way as things change we don't need to keep this message up to date.